No moar Dawn Patrol

burden opened the pull request:
Introducing a new Utensils! 

## new logo
![utensils-plain](https://user-images.githubusercontent.com/73517/61854731-8d80a080-ae73-11e9-967a-c188a78d07cb.png)

## home
![localhost_4051_ (1)](https://user-images.githubusercontent.com/73517/61854787-b012b980-ae73-11e9-98d0-c98c2c606949.png)

## blog
Additional settings for Categories and Tags are located in `src/_data/core.yml`

![localhost_4051_blog_ (1)](https://user-images.githubusercontent.com/73517/61854799-b739c780-ae73-11e9-83ac-271c504ba939.png)

## Service pages
Service and offer pages are generated by a plugin located at `src/_plugins/generate.rb` which reads from `src/_data/services.yml`.  

![localhost_4051_services_](https://user-images.githubusercontent.com/73517/61854894-e3eddf00-ae73-11e9-9287-f8277b13da1f.png)

![localhost_4051_services_software-engineering_](https://user-images.githubusercontent.com/73517/61854900-e7816600-ae73-11e9-9e2a-c477ebcf1eff.png)

## Offer pages
Offer pages should be saved in the following format `src/_includes/services/[SERVICE]/[OFFER].html`
![localhost_4051_services_software-engineering_back-end-development_](https://user-images.githubusercontent.com/73517/61855890-d33e6880-ae75-11e9-8720-72d67d67849c.png)

burden committed:
No moar Dawn Patrol

- new utensils logo
- blog
- service pages
- contact form

gemantzu commented:
Damn @burden , your templates are always so well done!!!

jamesbrink commented:
I know when I ran this locally the other day it was missing a favicon. we should probably scale down the logo and make sure we have that included as well

doomspork commented:
I noticed the same. I'll see about doing that this week and we can merge.

burden committed:
Adding work section + navbar refactor

burden commented:
Any thoughts on the direction I'm going with this favicon? I'll leave it up to @doomspork to determine whether it's missing a spork 😀 

![favicon-120x120](https://user-images.githubusercontent.com/73517/62335235-e659c580-b47f-11e9-8109-7bb486180f4a.png)

burden committed:
Adding favicons

burden committed:
Highlight Service in nav and footer

burden commented:
Removing the WIP tag. Thank you to everyone for the kind words 😍

Next steps will be added to issues as there are going to be a number of action items we'll need to address before we can finally launch.

burden committed:
update to jekyll 3.8.6

doomspork commented on src/_data/core.yml:
```diff
@@ -0,0 +1,54 @@
+      url: "/team/"
+  footer:
+    expertise: # todo: make this like services.yml
+      - caption: Docker
```
Docker / Kubernetes

doomspork commented on src/_data/core.yml:
```diff
@@ -0,0 +1,54 @@
+    expertise: # todo: make this like services.yml
+      - caption: Docker
+      - caption: Elixir / Phoenix
+      - caption: Ruby
```
Let's just make this: Ruby on Rails

That's what most people are looking for when they're Googling.

doomspork commented on src/_data/services.yml:
```diff
@@ -0,0 +1,51 @@
+  - name: Product Design # link caption
+    slug: digital-product-design # link url
+    title: Digital Product Design # page title
+    description: Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
```
Do we need to fill in a `description` for all of these in the file?

burden replied:
You can leave them blank, but since these descriptions pertain to SEO, I would have to advise against it. 

As a side note, this catalog is pretty much pulled directly from other agency sites. We should start completely fresh with our own catalog.

doomspork commented on src/_data/core.yml:
```diff
@@ -0,0 +1,54 @@
+    description: Pellentesque leo mauris, consectetur id ipsum sit amet, fergiat.
+projects:
+  - name: Culpa qui
+    repo: culpa_qui
```
Is there a way to support things other than repos?

burden replied:
Of course. What did you have in mind?

doomspork commented on src/_data/core.yml:
```diff
@@ -0,0 +1,54 @@
+    description: Pellentesque venenatis dolor imperdiet dolor mattis sagittis.
+tags:
+  - name: elixir
+    description: Donec urna ex, lacinia in purus ac, pretium pulvinar mauris.
```
Should we just copy something from their websites or wiki?

burden replied:
This description will be shown on archive pages for tags (all blog articles tagged "elixir"). 

You can certainly copy from a site if you want, but be careful. If it's verbatim, Google will know and ding you in search results for it.

burden committed:
Refactor services.yml

improved services.html
added cta to services.yml
added title and description to plugin pages using services.yml

burden commented:
Improved the Services page, and refactored services.yml a little. Checkout commit for details. 

![localhost_4051_services_(iPad Pro)](https://user-images.githubusercontent.com/73517/65309813-b48ce300-db41-11e9-97a8-46ec3517d95e.png)

burden committed:
Remove pages that won't launch from sitemap.xml
